integration-tests/sandbox-drain-delegate-operation.spec.ts

```diff
@@ -1,9 +1,9 @@
 import { TezosToolkit, Protocols } from "@taquito/taquito";
 import { CONFIGS, sleep } from "./config";
 
-CONFIGS().forEach(({ lib, protocol, rpc, setup, createAddress }) => {
+CONFIGS().forEach(({ lib, rpc, setup, createAddress }) => {
   const Tezos = lib;
-  const flextesanet = (protocol === Protocols.PtLimaPtL || protocol === Protocols.ProtoALpha) && rpc === 'http://localhost:20000' ? test : test.skip;
+  const flextesanet =  rpc === 'http://localhost:20000' ? test : test.skip;
```

Comment:
is there a reason these are using test rather than it? ive seen it in a bunch of these for certain testnets that it uses test rather than it

Reply:
no reason, they're both equivalent